Firestore/Source/Remote/FSTDatastore.m

```diff
@@ -557,21 +614,82 @@ - (void)resumeStartFromBackoff {
 
   // Momentarily set state to FSTStreamStateInitial as `start` expects it.
   self.state = FSTStreamStateInitial;
-  [self start];
+  [self startWithDelegate:delegate];
   FSTAssert([self isStarted], @"Stream should have started.");
 }
 
-- (void)stop {
-  FSTLog(@"%@ %p stop", NSStringFromClass([self class]), (__bridge void *)self);
+/**
+ * Closes the stream and cleans up as necessary:
+ *
+ * <ul>
```

Comment:
Nit: this isn't javadoc, so probably make this a markdown-list.

Reply:
Done.